thirdparty/intervaltree/IntervalTree.h

```diff
@@ -68,7 +68,7 @@ class IntervalTree {
         center = other.center;
         intervals = other.intervals;
         if (other.left) {
-            left = (intervalTree*) malloc(sizeof(intervalTree));
+            left = new intervalTree();
```

Comment:
not sure we should be changing code in thirdparty, unless forced to

Reply:
At least it should get fed back to upstream

Reply:
I agree, just jumped to the code and corrected. Returned.